src/main.cpp

```diff
@@ -299,6 +299,8 @@ static void set_allowed_options(OptionList *allowed_options)
 		_("Migrate from current players backend to another (Only works when using minetestserver or with --server)"))));
 	allowed_options->insert(std::make_pair("migrate-auth", ValueSpec(VALUETYPE_STRING,
 		_("Migrate from current auth backend to another (Only works when using minetestserver or with --server)"))));
+	allowed_options->insert(std::make_pair("migrate-mod-metadata", ValueSpec(VALUETYPE_STRING,
+		_("Migrate from current mod metadata backend to another (Only works when using minetestserver or with --server)"))));
```

Comment:
I think it would be more comprehensive if this option were called `mod storage` rather than `mod metadata` even if the current naming is technically correct.

Reply:
Should I also change the `mod_metadata_backend` config option?

Reply:
Yes, I think so for the same reason as above that "mod storage" is more known than "mod metadata".